docs/components/integration-framework/connectors/out-of-the-box-connectors/sendgrid.md

```diff
@@ -6,6 +6,9 @@ description: Quickly send emails from your BPMN processes.
 
 The **SendGrid Connector** allows you to quickly send emails from your BPMN processes.
 
+If you see that connector mark as 'Deprecated', you can update and replace it.
+See [Replacing Templates](../../../../components/modeler/desktop-modeler/element-templates/using-templates.md) for more.
+
```

Comment:
@Oleksiivanov same as above, no need to add these sentence to SendGrid Connector docs.
You can remove entire:

"If you see that connector mark as 'Deprecated', you can update and replace it.
See [Replacing Templates](https://github.com/components/modeler/desktop-modeler/element-templates/using-templates.md) for more."